resources/views/projects/milestones/create.blade.php

```diff
@@ -0,0 +1,82 @@
+@extends('layouts.app')
+
+@section('title', 'Create milestone')
+
+@section('content')
+
+<div class="padded content">
+	<h1>Create milestone</h1>
+
+	<form class="ui form {{ $errors->any() ? 'error': '' }}" method="post" enctype="multipart/form-data" action="">
+		@csrf
+
+		<!-- Milestone name -->
+		<div class="field">
+			<label for="milestoneName">Name</label>
+			<input type="text" name="milestoneName" id="milestoneName">
+		</div>
+
+		<!-- Previous milestone -->
+		<div class="field">
+			<label for="prevMilestone">Previous milestone</label>
+			<select class="ui fluid search dropdown" name="prevMilestone" id="prevMilestone">
+				{{-- @foreach ($milestones as $prevMilestone)
+					<option value={{ $prevMilestone->id }}> {{ $prevMilestone->name }} </option>
+				@endforeach --}}
+			</select>
+		</div>
+
+		<!-- Estimated date -->
+		<div class="field">
+			<label for="estimatedDate">Estimated date</label>
+			<input type="date" name="estimatedDate" id="estimatedDate">
```

Comment:
`date` type inputs have [horrible support](https://caniuse.com/#feat=input-datetime) let's use something else

Reply:
Check what @katiearriagam  is doing here:

https://github.com/ProyectoIntegrador2018/BlueMarket/blob/69e17de768d7580a97196ce7cbd8bb8de0a68300/resources/views/projects/tasks/create.blade.php#L28